src/coverage-system/coverageservice.ts

```diff
@@ -179,5 +180,12 @@ export class CoverageService {
         window.showWarningMessage(message.toString());
         this.outputChannel.appendLine(`[${Date.now()}][gutters]: Error ${message}`);
         this.outputChannel.appendLine(`[${Date.now()}][gutters]: Stacktrace ${stackTrace}`);
+
+        const crashReporter = new CrashReporter();
+
+        if (crashReporter.checkEnabled()) {
+            const [ sentryId, sentryPrompt ] = crashReporter.captureError(error);
```

Comment:
> Thanks again for the contribution, this is looking great!
> I can do some QA after this is merged to the mainline branch and it will then go out in 2.7.0

My pleasure. Thank you for guiding me through it